src/seat/touch.rs

```diff
@@ -38,7 +38,9 @@ pub trait TouchHandler: SeatHandler + Sized {
     ///
     /// The associated touch ID ceases to be valid after the touch up event with the associated ID
     /// and may be reused for other touch points after that.
-    #[allow(clippy::too_many_arguments)]
+    ///
+    /// Coordinates are surface-local.
+    #[warn(clippy::too_many_arguments)]
```

Comment:
I don't think we need to keep that

Reply:
I mean, this lint is already warn by default, why explicitly enable it again?

Reply:
Ah, I meant to `allow` it, not `warn` on it.